lawnchair/res/drawable/ic_yandex.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="#ffffff"
+        android:pathData="M9.957,0.199C12.32,-0.215 14.805,0.105 16.984,1.105C19.578,2.277 21.715,4.41 22.891,7C24.059,9.516 24.285,12.445 23.57,15.117C22.777,18.117 20.762,20.766 18.082,22.324C15.383,23.934 12.039,24.387 9.004,23.598C6.219,22.891 3.711,21.137 2.102,18.754C0.375,16.246 -0.32,13.055 0.184,10.055C0.547,7.793 1.582,5.645 3.129,3.953C4.895,1.984 7.348,0.641 9.957,0.199ZM14.613,12.809L17.434,6.402L15.09,6.402L12.27,12.809ZM13.109,19.656L13.109,17.406C13.109,15.707 12.898,15.211 12.25,13.801L8.855,6.402L6.453,6.402L10.801,15.859L10.801,19.656ZM10.383,2.191C7.047,2.711 4.086,5.074 2.816,8.199C1.836,10.52 1.797,13.215 2.719,15.559C3.703,18.148 5.824,20.289 8.418,21.27C10.648,22.156 13.203,22.168 15.445,21.324C18.023,20.383 20.172,18.316 21.199,15.77C21.992,13.875 22.152,11.727 21.684,9.73C21.152,7.477 19.816,5.414 17.949,4.043C15.824,2.434 13.016,1.723 10.383,2.191ZM10.383,2.191" />
+</vector>
```

Comment:
While the icon looks fine when themed icons are on, it is illegible when in light mode without themed icons.

To get the best of both icons, I'd suggest moving this to a file `ic_yandex_tinted.xml`, while the previous icon file can be applied here. See also the comment to the `Yandex.kt` file below.